### PR TITLE
Fix Clickable hyperlinks in youtube chat

### PR DESCRIPTION
**Pull request by Excellify**
Renders links as clickable when the sender is a moderator in the channel.

### CODE REVIEW

**Review by AnatoleAM — CHANGES_REQUESTED** [review]
One small problem

File: src/Sites/youtube.com/Runtime/Tokenizer.ts [hunk 1]
```diff
@@ -74,6 +72,22 @@ export class Tokenizer {
 		ctx.appendChild(span);
 	}
 
+	/**
+	 * Append a text part to the new message context
+	 */
+	addHyperlinkPart(ctx: HTMLSpanElement, text: string): boolean {
```
Comment (AnatoleAM) [inline src/Sites/youtube.com/Runtime/Tokenizer.ts:78]: This doesn't  open in a new tab, so if a user clicks a link they redirect to the page and quit the stream

**Review by AnatoleAM — CHANGES_REQUESTED** [review]

File: src/Sites/youtube.com/Runtime/Tokenizer.ts (hunk 1 above)
Comment (Excellify) [reply src/Sites/youtube.com/Runtime/Tokenizer.ts:78]: Oops, ill fix that

**Review by AnatoleAM — APPROVED** [review]
👍

**Pull request merged**